app/celery/letters_pdf_tasks.py

```diff
@@ -262,6 +265,100 @@ def process_virus_scan_passed(self, filename):
         update_notification_status_by_id(notification.id, NOTIFICATION_TECHNICAL_FAILURE)
 
 
+@notify_celery.task(bind=True, name='sanitise-letter', max_retries=15, default_retry_delay=300)
+def sanitise_letter(self, filename):
+    try:
+        reference = get_reference_from_filename(filename)
+        notification = dao_get_notification_by_reference(reference)
+
+        current_app.logger.info('Notification ID {} Virus scan passed: {}'.format(notification.id, filename))
+
+        if notification.status != NOTIFICATION_PENDING_VIRUS_CHECK:
+            current_app.logger.exception('Sanitise letter called for notification {} which has is in {} state'.format(
```

Comment:
Why are we logging an exception here? Isn't it ok if the notification has already been marked as `created`?

Reply:
If the notification is already marked as `created` then the task has already run - I had assumed that that was an issue and that we'd want to know about it, but if that's not the case then I can change it to a different level of log message

Reply:
I've downgraded this log message and other similar one from `exception` to `info`